modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install

```diff
@@ -381,23 +381,22 @@ function social_group_flexible_group_update_8912() {
  * Allow CM+ translate "Flexible Group" group type.
  */
 function social_group_flexible_update_10301() {
-  foreach (['sitemanager', 'contentmanager'] as $role_id) {
-    user_role_grant_permissions($role_id, ['translate flexible_group group']);
-  }
+  // Removed hook due to that it doesn't apply since wrong current function
```

Comment:
`//s` 

Removed code in the function as it is neither an update hook nor it is called anywhere. The name of the function should have been `social_group_flexible_group_update_10301() ` which qualifies it to be an update hook. We have moved the code too a new update hook().

Reply:
Updated